src/EnergyPlus/HVACVariableRefrigerantFlow.cc

```diff
@@ -8663,34 +8663,35 @@ namespace HVACVariableRefrigerantFlow {
         }
 
         // Set inlet air mass flow rate based on PLR and compressor on/off air flow rates
-        SetAverageAirFlow(VRFTUNum, 1.0, temp);
+        //SetAverageAirFlow(VRFTUNum, 1.0, temp);
```

Comment:
The TUs have already been simulated. At this point it appears the model want to know what the evap T and cond T are for use during the next time step. Anything that tried to resimulate the TUs is now commented out.

Reply:
Note here that the original model did not set OA flow rate and therefore used the OA flow for the last TU simulated when SetAverageAirFlow was called. When I corrected this (lines 8671 and 8676), my new method matched the old method results.

Reply:
Also note that any calls to this and other functions updates the node flow rates. So data set when the TU was actually modelled is used here instead.